WordPressKitTests/JetpackProxyServiceRemoteTests.swift

```diff
@@ -0,0 +1,123 @@
+import XCTest
+@testable import WordPressKit
+
+class JetpackProxyServiceRemoteTests: XCTestCase {
+    let timeout: TimeInterval = 1.0
+    let api = MockWordPressComRestApi()
+    let siteID = 1001
+
+    private var remote: JetpackProxyServiceRemote {
+        print(">>> CALL: \(api)")
```

Comment:
Nitpick: Is this intended to print here? I'm assuming it was for [this](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/576/files#r1099041223) comment. I guess it isn't a huge deal though since it's just tests.

Reply:
Good catch! Addressed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/576/commits/4e6044766ab6f782a6662355aa87b6528030ce5a.